docs/course_materials/variables_data_types.html

```diff
@@ -389,9 +389,9 @@ <h3><b>Formatting</b></h3>
               <pre><code data-trim data-noescape class="hljs lang-python">
     >>> name = 'Tom'
     >>> family_name = 'Sawyer'
-    >>> print('Hello %s!' % name)
+    >>> print(f'Hello {}!')
```

Comment:
shouldn't it be? : print(f'Hello {name}!')

Reply:
Corrected the statement.